Update arweave-js dep to ^1.6.0 and misc small fixes.

aidanok opened the pull request:
Redid this PR to only be bugfix, will work as is but to get the full 10MB tx supported, 
needs:  https://github.com/ArweaveTeam/arweave-js/pull/30 

Will work without, but files somewhere above 7MB will generate an axios error.

aokisok commented:
Just to give a bit more context about the limit to 7MB: 

axios has an option maxRequestBody size. 

This defaults to 10MB ( I believe ) but there's an issue in older versions
where changing this his did not work correctly:  
https://github.com/axios/axios/issues/1286

Combined with base64 encoding, uploading a file with `deploy-dir` that is 8MB on disk
will fail with `Request body larger than maxBodyLength limit`. 

The full fix is to update the arweave-js dependency in arweave-deploy to the latest,
which uses a newer version of axios, and also set the axios `maxRequestBody` size
to something bigger to account for base64 encoding.

aokisok commented:
One more comment: 

- Technically its a breaking change, since although 8MB files failed before, the exact size that failures occured was somewhere between 7MB & 8MB, depending on tag size etc, so someone might have had a 7.2MB file that deployed ok but would fail with this version. 

Just made this PR as someone in Discord wasn't able to use `deploy` with more than 2MB and I had previously investigated the axios issue. 
  
May be better to wait and fix the dependencies & axios settings so the full 10MB is supported.

aidanok committed:
arweave-js ^1.6.0, `deploy` 10MB, bump semver patch

This needed esModuleInterop turned on to update arweave-js
to ^1.6.0, the modules imported with @ts-ignore are checked
and fixed where needed.

- Fixes 2MB limit for `deploy` command
- Correct error message.
- Allows update of arweave-js to allow full 10MB tx size.

arweave-kyle committed:
Merge branch 'master' into master
